src/postprocessors/BetaEffective.C

```diff
@@ -74,17 +74,18 @@ BetaEffective::getValue() const
 
   switch (_output)
   {
-    case kinetics::KineticsOutputEnum::Value:
+    case eigenvalue::EigenKineticsOutputEnum::Value:
       return beta_eff;
 
-    case kinetics::KineticsOutputEnum::StDev:
+    case eigenvalue::EigenKineticsOutputEnum::StDev:
```

Comment:
The use of `eigenvalue` as the enumeration is kind of confusing to someone reading the code - can be name it something like `statistics::OutputEnum`?